controllers/provider-aws/pkg/apis/aws/validation/infrastructure_test.go

```diff
@@ -246,7 +356,31 @@ var _ = Describe("InfrastructureConfig validation", func() {
 
 			Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
 				"Type":  Equal(field.ErrorTypeInvalid),
-				"Field": Equal("networks"),
+				"Field": Equal("networks.vpc"),
+			}))))
+		})
+
```

Comment:
should we also add test for allowing the addition of zones while forbidding changing old ones (both happening at the same time)?

Reply:
Good idea. Test has been added.